_notebooks/2020-12-12-Get-and-Extract-Data.ipynb

```diff
@@ -24,7 +24,8 @@
     "# hide\n",
```

Comment:
<p>The type hint for <code>file</code>  is <code>Path</code> while the open syntax is using the <code>str</code> representation of the same. We should either <code>file.open()</code> in the <code>Pathlib</code> syntax or change the type hint. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/projectnoor/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>